app/constant.ts

```diff
@@ -53,7 +53,13 @@ export const EXPORT_MESSAGE_CLASS_NAME = "export-markdown";
 export const OpenaiPath = {
```

Comment:
> **Note**
  > For other developers, if you choose to use a different GPT model or any other alternative to the official OpenAI models, it is crucial to ensure this text moderation support or not. This is important for the future. 
Otherwise try made own path

Reply:
key authnya apa om

Reply:
yang itu cmn buat production/demo buat kontribusi